src/auth/server/local.ts

```diff
@@ -21,7 +21,7 @@ export function createDisposableAuthSever() {
             app.get('/callback', (request, response) => {
                 const { access_token: accessToken, refresh_token: refreshToken, error } = request.query;
                 if (!error) {
-                    res({ accessToken, refreshToken });
+                    res({ accessToken: accessToken as string, refreshToken: refreshToken as string });
```

Comment:
You could just do this:

```suggestion
                    res({ accessToken, refreshToken });
```

or this but the "casting" here is superfluous.

```suggestion
                    res(<CreateDisposableAuthSeverPromiseResult>{ accessToken, refreshToken });
```